src/header/shared/httpdate.rs

```diff
@@ -46,12 +46,7 @@ impl FromStr for HttpDate {
 
 impl Display for HttpDate {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        let tm = self.0;
-        let tm = match tm.tm_utcoff {
```

Comment:
Unless this was fixed in the time crate, `to_utc` is slow even if the time is already in UTC.